Jenkinsfile.dev

```diff
@@ -112,12 +124,68 @@ if(hasRepoChanged){
               type: "button",
               text: "View Build Logs",
               style:"danger",        
-              url: "${currentBuild.absoluteUrl}/console"
+              APP_URL: "${currentBuild.absoluteAPP_URL}/console"
             ]
           ])
       }
     }
   }
+
+  podTemplate(label: 'bddstack', name: 'bddstack', serviceAccount: 'jenkins', cloud: 'openshift', containers: [
+    containerTemplate(
+      name: 'jnlp',
+      image: '172.50.0.2:5000/openshift/jenkins-slave-bddstack',
```

Comment:
Shouldn't the image reference come from the ENV? i.e. `env.JENKINS_BDD_IMAGE`? 
Or at least remove specific IP addresses from configuration and replace with env vars; e.g. env.CLUSTER_IP_ADDRESS or similar...

Reply:
Let me ask for more info and get back to you on this as I copied the step over from GWells.